Source/Plugins/Core/com.equella.core/js/tsrc/search/components/SearchResult.tsx

```diff
@@ -0,0 +1,241 @@
+    event.stopPropagation();
+    setAttachExpanded(!attachExapanded);
+  };
+
+  const thumbnail = (disableThumb: boolean) => {
+    if (attachments.length > 0 && !disableThumb) {
+      return (
+        <>
+          <img
+            className={classes.thumbnail}
+            src={attachments[0].links.thumbnail}
+            alt={attachments[0].description}
+          />
+        </>
+      );
+    } else {
+      return (
+        <Subject className={classes.placeholderThumbnail} fontSize="large" />
+      );
+    }
+  };
+
+  const itemMetadata = (
+    <div className={classes.additionalDetails}>
+      <Typography component="span" variant="body1" className={classes.status}>
+        {status}
+      </Typography>
+      <Divider
+        flexItem
+        component="span"
+        variant="middle"
+        orientation="vertical"
+      />
+      <Typography component="span" variant="body1">
+        {searchResultStrings.dateModified}&nbsp;
+        <DateDisplay displayRelative date={new Date(modifiedDate)} />
+      </Typography>
+    </div>
+  );
+
+  const customDisplayMetadata = displayFields.map((element: any) => {
+    return (
+      <>
+        <ListItem disableGutters dense>
+          <Typography
+            component="span"
+            variant="body2"
+            className={classes.heading}
+            color="textPrimary"
+          >
+            {element.name}
+          </Typography>
+          <Typography
+            component="span"
+            variant="body2"
+            className={classes.inline}
+            color="textPrimary"
+            dangerouslySetInnerHTML={{ __html: element.html }}
+          ></Typography>
+        </ListItem>
+      </>
+    );
+  });
+
+  const generateAttachmentList = () => {
+    const attachmentsList = attachments.map((attachment: any) => {
+      return (
+        <>
+          <ListItem
+            key={attachment.id || attachment.uuid}
+            button
+            className={classes.nested}
+          >
+            <ListItemIcon>
+              <InsertDriveFile />
+            </ListItemIcon>
+            <ListItemText color="primary" primary={attachment.description} />
+          </ListItem>
+        </>
+      );
+    });
+
+    if (attachmentsList.length > 0)
+      return (
+        <List component="div">
+          <ListItem
+            button
+            onClick={(event) => handleAttachmentPanelClick(event)}
+          >
+            <ListItemIcon>
+              <AttachFile />
+            </ListItemIcon>
+            <ListItemText primary={searchResultStrings.attachments} />
+            {attachExapanded ? <ExpandLess /> : <ExpandMore />}
+          </ListItem>
+          <Collapse in={attachExapanded} timeout="auto" unmountOnExit>
+            <List component="div" disablePadding>
+              {attachmentsList}
+            </List>
+          </Collapse>
+        </List>
+      );
+    else return null;
```

Comment:
the `if` contains a `return`, the `else` on the line can be omitted.

```suggestion
    return null;
```

Reply:
I didn't know that. That's very good to know